### ISSUE TITLE
Can't put the app in favorite

### ISSUE THREAD

**Issue by florentdestremau** [aunetx/deezer-linux#8]
![image](https://user-images.githubusercontent.com/2852204/150323822-7f9cbea0-9b10-4e66-ac34-d94fd309d32d.png)
The icon in the app bar does not offer "Add to favorites" option, so I can't launch it from there, I have to search for it in all applications.

**Comment by aunetx** [comment]
Which version are you using? Appimage maybe? And you are on gnome right?

**Comment by florentdestremau** [comment]
- Ubuntu 21.10 - Gnome
- `.deb` installation
- v5.30.130-1 (installed 2 days ago)

**Comment by joenepraat** [comment]
I can confirm this on Ubuntu 20.04 LTS as well. I can add it to the favorites with via search or view applications (right click), but than when I launch Deezer it will create another icon in my dock (see screenshot).

![afbeelding](https://user-images.githubusercontent.com/41594439/153038105-e60d2d8c-bde2-4ecf-9166-8b9965920699.png)

**Comment by florentdestremau** [comment]
@aunetx do you think that an install from another package manager may change this ?

**Comment by aunetx** [comment]
No, I don't think so -- it actually does the same for me, using Flatpak or rpm packages.
I will see what I can do about it, I think I can find something in `electron-builder` documentation (as it does the actual packaging, although this may has something to do with deezer directly)
